gh-md-toc

```diff
@@ -235,23 +235,23 @@ gh_toc_grab() {
                          }
                          modified_href = modified_href res
                     }
-                    print sprintf("%*s", (level-1)*'"$2"', "") "* [" text "](" gh_url  modified_href ")"
+                    print sprintf("%*s", (level)*'"$2"', "") "* [" text "](" gh_url  modified_href ")"
```

Comment:
this change will add an indention event for the first line in the TOC. I believe we do not need to add it. 
```suggestion
                    print sprintf("%*s", (level-1)*'"$2"', "") "* [" text "](" gh_url  modified_href ")"
```